### PR TITLE
PRO-183 upgrade Play 2.5

### PR DESCRIPTION
**Pull request by mgarciaberlin**
Ticket: https://advancedtelematic.atlassian.net/browse/PRO-183

Follow-up to #441

### CODE REVIEW

**Comment by mgarciaberlin** [comment]
The `APIFunTests` tests are flaky on CI. After two or three attempts, they are green.

File: web-server/app/org/genivi/webserver/controllers/Application.scala [hunk 1]
```diff
@@ -76,7 +80,12 @@ class Application @Inject() (ws: WSClient, val messagesApi: MessagesApi, ldapAut
     wreq.execute.map { resp =>
       Result(
         header = ResponseHeader(resp.status, resp.allHeaders.mapValues(x => x.head)),
-        body = Enumerator(resp.bodyAsBytes))
+        body = {
+          val bstr = resp.bodyAsBytes
+          val blen = bstr.length.toLong
+          play.api.http.HttpEntity.Streamed(Source.single(bstr), contentLength = Some(blen), contentType = None)
```
Comment (simao) [inline web-server/app/org/genivi/webserver/controllers/Application.scala]: Isn't `resp.bodyAsBytes` a `Source`? It's strange to wrap some bytes in a `Source.single`, `asBytes` should be some kind of source. is there some conversion from a request body to a Source[Byte]  ?

Comment (simao) [inline web-server/app/org/genivi/webserver/controllers/Application.scala]: I think for this you can use `wreq.stream().execute`, then you'll get a `resp: StreamedResponse` which you can pass on as response?

Comment (mgarciaberlin) [inline web-server/app/org/genivi/webserver/controllers/Application.scala]: Good idea.

**Comment by simao** [comment]
:+1: :ship: 

Also gets rid of that weird thing of specifying length for a streamed response :)

**Comment by simao** [comment]
I am worried about those flaky tests on ci though, they are not flaky in other branches...

**Pull request merged**